string name
string message
string hardware_id
system_monitor/GPUStatus status
